standard/localisation.lua

```diff
@@ -26,7 +26,7 @@ function Localisation.getCountryName(country, noentry)
 	return countryname
 end
 
-function Localisation.getLocalisation(country, noentry)
+function Localisation.getLocalisation(country, noentry, simpleError)
```

Comment:
```suggestion
function Localisation.getLocalisation(country, noentry, shouldReturnSimpleError)
```

Would probably be better to use a table called `options` for this though:
```lua
Localisation.getLocalisation(country, {shouldReturnSimpleError = true})
```

(I can't quite figure out what the purpose of `noentry` is, but seems like that should be inside the table too)

Reply:
i copied it from the existing module
it suppresses the error msg and category
could rename it to `noErrorDisplay` or `noErrorCategory` or something like that

Reply:
Direct invoke from wikicode:

```
{{#invoke:Localisation|getLocalisation|'Netherlands'|shouldReturnSimpleError=true}}
```